src/stratigraphy/util/find_depth_columns.py

```diff
@@ -37,27 +31,51 @@ def value_as_float(string_value: str) -> float:  # noqa: D103
                 value = value_as_float(match.group(1))
                 entries.append(DepthColumnEntry(word.rect, value))
             elif include_splits:
-                # support for e.g. "1.10-1.60m" extracted as a single word
-                regex2 = re.compile(r"^-?([0-9]+(\.[0-9]+)?)[müMN\\.]*\W+([0-9]+(\.[0-9]+)?)[müMN\\.]*$")
-                match2 = regex2.match(input_string)
-
-                if match2:
-                    value1 = value_as_float(match2.group(1))
-                    first_half_rect = fitz.Rect(
-                        word.rect.x0, word.rect.y0, word.rect.x1 - word.rect.width / 2, word.rect.y1
-                    )
-                    entries.append(DepthColumnEntry(first_half_rect, value1))
-
-                    value2 = value_as_float(match2.group(3))
-                    second_half_rect = fitz.Rect(
-                        word.rect.x0 + word.rect.width / 2, word.rect.y0, word.rect.x1, word.rect.y1
-                    )
-                    entries.append(DepthColumnEntry(second_half_rect, value2))
+                entries.extend(extract_layer_depth_interval_entries(input_string, word))
         except ValueError:
             pass
     return entries
 
 
+def value_as_float(string_value: str) -> float:  # noqa: D103
+    # OCR sometimes tends to miss the decimal comma
+    parsed_text = re.sub(r"^-?([0-9]+)([0-9]{2})", r"\1.\2", string_value)
+    return abs(float(parsed_text))
+
+
+def extract_layer_depth_interval_entries(
+    input_string: str, line: TextWord | TextLine, require_start_of_string: bool = True
+) -> list[DepthColumnEntry]:
+    """Extracts two DepthColumnEntry obejcts from a string to represent a layer depth interval.
+
+    Args:
+        input_string (str): The string to extract the depth interval from.
+        line (TextLine): The line object containing the rectangle of the string.
```

Comment:
Types in the docstring don't match. Also, the value of `input_string` immediately gets overridden. And it can be confusing that `line` can be both a `TextLine` as well as a `TextWord`. 

Wouldn't it be sufficient to accept params `text: str` and `rect: fitz.Rect` instead of `input_string` and `line`.

Or alternatively, if we really want to have params that can be both `TextLine` as well as `TextWord`, then it would be cleaner to make them children of a common abstract parent class.